public/stylesheets/error_page.css

```diff
@@ -14,7 +14,7 @@ h1 {
 }
 
 #header {
-    background-image: url("/assets/loomio-orange.png");
+    background-image: url("/theme/large_logo.png");
     background-repeat: no-repeat;
     height: 150px;
     text-indent: -999em;
```

Comment:
I think the 'mascot.gif' is the wrong one to move (I don't believe we use that asset anywhere)... we want 'public/img/mascot.png' instead (which is used in the oauth apps and for the auth modal and slack pages). I'd also be happy to do this without customizing the mascot for now.

Reply:
yea, I backtracked on that and hoped you'd spot where I did not clean up properly

Reply:
so well done